### PR TITLE
tests/gnrc_tcp_*: remove default PORT for native

### PR DESCRIPTION
**Pull request by PeterKietzmann**
### Contribution description

This PR removes the default `PORT?=` which was set to run the TCP client/server test applications on two native instances. A problem occurs when calling `BOARD=xyz make flash term` (with xyz not native) because in that case, `PORT` is set to `tap` by defualt. 

### Testing procedure

Build/flash/term a board with one of the applications *tests/gnrc_tcp_** with and without this PR.

### Issues/PRs references
 #10947

### CODE REVIEW

**Comment by miri64** [comment]
The point of those is, I believe that those two apps belong to each other. IMHO we need to rework these tests in general (however, having the first bisect in https://github.com/RIOT-OS/RIOT/issues/10947 having a Python script interacting via a TCP socket with the test is probably not a good solution either)

File: tests/gnrc_tcp_client/README.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ The test sequence above runs a configurable amount of times.
 Usage (native)
 ==========
 
-Build and run test:
-make clean all term
+Build and run test (don't forget to use different tap device than server):
+make clean all term PORT=tap1
```
Comment (smlng) [inline tests/gnrc_tcp_client/README.md]: such a line should be added to the servers readme, too ... or not?

Comment (miri64) [reply tests/gnrc_tcp_client/README.md]: `PORT` defaults to `tap0` so it is not strictly necessary.

Comment (smlng) [reply tests/gnrc_tcp_client/README.md]: alternatively, as some might overlook this or do not know that `tap0` is default, we could add:

```
ifeq (native,$(BOARD))
PORT ?= tap1
endif
```

in the Makefile instead

**Comment by PeterKietzmann** [comment]
Fine with me. Changed, squashed, pushed...

**Comment by smlng** [comment]
@PeterKietzmann I assume (and trust) that you've tested this 😄

**Comment by PeterKietzmann** [comment]
@smlng you should never do that :-P! However, I was able to start both programs via `make all term`. The server application automatically chose `tap0`, the client chose `tap1`.

**Comment by miri64** [comment]
Confirmed.

**Pull request merged**